tests/daemon_test.py

```diff
@@ -8,7 +8,7 @@
 
 import pytest
 
-from . constants import *
+from . import constants
```

Comment:
could import as alias:
from . import constants as const

then use the short name:
const.DETLA_DISK_MAGIC
etc.

Reply:
We have only few constants and their names is pretty clear, so this change does not improve the code.

Lets leave the constants as they are used today.